tests/test_core.py

```diff
@@ -432,11 +432,22 @@ def test_calculate_squared_distance_profile(Q, T):
         )
         ** 2
     )
+
+    Q_subseq_isconstant = core.rolling_isconstant(Q, m)
+    T_subseq_isconstant = core.rolling_isconstant(T, m)
     QT = core.sliding_dot_product(Q, T)
     μ_Q, σ_Q = core.compute_mean_std(Q, m)
     M_T, Σ_T = core.compute_mean_std(T, m)
+
     comp = core._calculate_squared_distance_profile(
-        m, QT, μ_Q.item(0), σ_Q.item(0), M_T, Σ_T
+        m,
+        QT,
+        μ_Q.item(0),
```

Comment:
Sure. Thank you for the suggestion!

Just one question:

Since we are using `μ_Q` and `σ_Q` only once in this test function, I was wondering if we could do:
```
μ_Q, σ_Q = core.compute_mean_std(Q, m)
```

And then pass `μ_Q[0]` and `σ_Q[0]` to the function. 

What do you think?